misc/README.md.erb

```diff
@@ -586,6 +586,22 @@ end
 
 On this case, you can see the result of `bar()` every time you stop there.
 
+You can use `pre` and `do` together to create more useful workflows.
```

Comment:
I'm not sure why this added block is needed.

Reply:
It wasn't very obvious to me, how I would achieve something like this. I think the technique used is handy in alot of situations but as I said, it wasn't obvious how I would go about doing it. So, I think it's useful to have this documented here.

I would have never thought that 
```
binding.b do: 'watch @a ;; break foo pre: info'
```
the `pre` inside the string above would be evaluated correctly, like an argument to a `binding.b` and thus suspend the program. I would have never thought this would work, so I think it's worth documenting. 

But If I missed something in the README that does suggest that this is possible and I missed it, please let me know :smile:

Reply:
I'm curious why it is not clear without explanation. `do:` accepts debug command string (described) and `break foo pre: info` is debug command.

on this case, 

```ruby
debugger do: 'watch @a'
debugger do: 'break foo pre: info'

# or
debugger do: <<~DO
  watch @a
  break foo pre: info
DO
```

is more readable for me.
(I usually use `;;` for writability/easy to write, though)

Reply:
I updated my PR removing the above block and I added a sentence about how users can use `;;` instead of `\n`.
I think this will do, no?
Let me know if you want me to add something else.

Also hope `;;` stays around :smile: